tests/manualTests/html/keyboardInput-manualTests.html

```diff
@@ -21,8 +23,14 @@
         <p><a id="gpiic-enable-sticky-keys" href="#">Enable Sticky Keys</a></p>
         <p><a id="gpiic-disable-sticky-keys" href="#">Disable Sticky Keys</a></p>
         <script>
-            fluid.defaults("gpii.tests.firstDiscovery.keyboardInput", {
-                gradeNames: ["gpii.firstDiscovery.keyboardInput", "autoInit"],
+            fluid.registerNamespace("gpii.tests.firstDiscovery.keyboardInput");
+            gpii.tests.firstDiscovery.keyboardInput.logInputChange = function (userInput) {
+                console.log(userInput);
```

Comment:
I know that we'll probably remove this file once FLOE-299 is added, but in the meantime we should probably use fluid.log here instead.